boot/env.ml

```diff
@@ -59,15 +61,15 @@ let fail_lib lib =
   eprintf "File not found: %s\n" lib;
   eprintf "The path for Coq libraries is wrong.\n";
   eprintf "Coq prelude is shipped in the rocq-core package.\n";
-  eprintf "Please check the COQLIB env variable or the -coqlib option.\n";
+  eprintf "Please check the ROCQLIB env variable or the -coqlib option.\n";
```

Comment:
I guess, we'll need a further PR to add a `-rocqlib` option and deprecate `-coqlib`.